src/test/java/seedu/address/model/person/AttendanceTest.java

```diff
@@ -70,4 +79,14 @@ public void constructor_invalidDate_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> new Attendance(invalidMonth));
     }
 
+    @Test
+    public void equals() {
+        Attendance testAtt = new Attendance("2022-02-02");
+        // to check self
+        assertEquals(testAtt, testAtt);
+        // to check with other commands
+        assertFalse(testAtt.equals(new Homework("homework")));
+
+    }
```

Comment:
No problem here, but maybe since for other equals we test against things like null, can add those too?

Reply:
Can can, have made the necessary changes